app/src/main/java/org/oppia/app/help/HelpItems.kt

```diff
@@ -0,0 +1,15 @@
+package org.oppia.app.help
+
+/** Enum class containing the items for the Recycler view of Help Activity */
```

Comment:
Update:
`/** Enum class containing the items for the Recycler view of [HelpActivity]. */`